client/components/Login.js

```diff
@@ -37,8 +35,8 @@ const LoginForm = (props) => {
   );
 };
 
```

Comment:
did you add line 38 and remove line 41? if we're keeping the big block of note, i feel like here were can just use the multi-line comment. i don't think we should use both a single-line comment and multi-line comment in one spot.

Reply:
Having a hard time seeing this one. We can chat more about this on Monday.